js/modules/auto_orient_image.js

```diff
@@ -0,0 +1,45 @@
+const loadImage = require('blueimp-load-image');
+
+const DEFAULT_JPEG_QUALITY = 0.85;
+
+// File | Blob | URLString -> LoadImageOptions -> Promise<DataURLString>
+//
+// Documentation for `options` (`LoadImageOptions`):
+// https://github.com/blueimp/JavaScript-Load-Image/tree/v2.18.0#options
+exports.autoOrientImage = (fileOrBlobOrURL, options = {}) => {
+  const { quality } = options;
+  if (typeof quality !== 'undefined' && typeof quality !== 'number') {
```

Comment:
We could use lodash's `isUndefined()` and `isNumber()` here instead of doing it manually. What do you think?

Reply:
👍 for future cases! In this case, I decided to kill the check altogether 👀